content_scripts/vimium_frontend.coffee

```diff
@@ -39,15 +39,20 @@ bgLog = (args...) ->
 # If an input grabs the focus before the user has interacted with the page, then grab it back (if the
 # grabBackFocus option is set).
 class GrabBackFocus extends Mode
-  constructor: ->
+  constructor: (isUrlChange) ->
     super
       name: "grab-back-focus"
+      singleton: "grab-back-focus"
       keydown: => @alwaysContinueBubbling => @exit()
 
     @push
       _name: "grab-back-focus-mousedown"
       mousedown: => @alwaysContinueBubbling => @exit()
 
+    # If it's just that the URL has changed, then bail if an input is already focused.  On some pages (e.g.
+    # search on Google Inbox) it's the act of focusing the input which triggers the URL change.
+    @exit() if isUrlChange and document.activeElement and DomUtils.isEditable document.activeElement
```

Comment:
I cann't understand this line. Here `document.activeElement` should still be an `<a>` node, shouldn't it?

For an `<input>` clicked, `return true if DomUtils.isFocusable document.activeElement` at line 78 should have stopped.